app/components/Layout.tsx

```diff
@@ -17,7 +17,7 @@ const StyledLayout = styled('div')`
   min-height: 100vh;
 `;
 const StyledMain = styled('main')`
-  margin: auto 50px;
+  margin: 50px auto;
```

Comment:
Why is this changed? This is making pages such as organization location very narrow. If there are overall layout fixes to make, they belong in a separate PR

Reply:
oh that was a mistake, I was playing with something because I noticed the layout isn't centered anymore. I guess our happo setup doesn't catch things like that. I'll remove it though.

Reply:
Happo did catch it, but we have so many spurious diffs that we're blindly approving happo currently